flink-runtime/src/test/java/org/apache/flink/runtime/rest/handler/legacy/ExecutionGraphCacheTest.java

```diff
@@ -59,7 +59,7 @@
 import static org.junit.Assert.fail;
 
 /**
- * Tests for the {@link ExecutionGraphCache}.
+ * Tests for the {@link DefaultExecutionGraphCache}.
  */
 public class ExecutionGraphCacheTest extends TestLogger {
```

Comment:
rename to `DefaultExecutionGraphCacheTest`?

Reply:
Makes sense. I will change it.